folly/io/async/AsyncServerSocket.cpp

```diff
@@ -212,7 +212,7 @@ int AsyncServerSocket::stopAccepting(int shutdownFlags) {
   // removeAcceptCallback().
   std::vector<CallbackInfo> callbacksCopy;
   callbacks_.swap(callbacksCopy);
-  for (std::vector<CallbackInfo>::iterator it = callbacksCopy.begin();
+  for (auto it = callbacksCopy.begin();
```

Comment:
This change is fine for this diff, since naming the type explicitly here likely comes from a time before `auto` and is likely not the intention. But this loop could, in a separate PR, be changed to a range-for.

Reply:
I believe that is actually done. But I'm splitting it up by parts for easier review.